app/models/weam.rb

```diff
@@ -63,7 +63,9 @@ class Weam < ActiveRecord::Base
     'distance learning' => { column: :distance_learning, converter: BooleanConverter },
     'priority enrollment' => { column: :priority_enrollment, converter: BooleanConverter },
     'preferred provider' => { column: :preferred_provider, converter: BooleanConverter },
-    'stem indicator' => { column: :stem_indicator, converter: BooleanConverter }
+    'stem indicator' => { column: :stem_indicator, converter: BooleanConverter },
+    'campus type' => { column: :campus_type, converter: BaseConverter },
```

Comment:
Should we add a validation for Campus Type?
Should we raise an error if it's ever blank, or a value other than Main, Branch or Extension?